include/grpcpp/support/proto_buffer_reader.h

```diff
@@ -154,7 +154,7 @@ class ProtoBufferReader : public grpc::protobuf::io::ZeroCopyInputStream {
       }
       uint64_t slice_length = GRPC_SLICE_LENGTH(*slice());
       set_byte_count(ByteCount() + slice_length);
-      if (slice_length <= count) {
+      if (slice_length <= static_cast<uint64_t>(count)) {
```

Comment:
I feel a better fix is to change the signature of this function to something like `virtual bool ReadCord(absl::Cord* cord, uint64_t count)`, since `count` should never go below 0.

I will let the author @veblush to comment on this.

Reply:
From what I see, `ReadCord(absl::Cord* cord, int count)` is a virtual method defined in a Protobuf (https://github.com/protocolbuffers/protobuf/blob/a01d047456783b46a4ca675eee881de110a88799/src/google/protobuf/io/zero_copy_stream.h#L194), and in grpc you override it, so unless this change of type is made directly in protobuf, the method's signature has to keep using `int`. And in protobuf it seems the whole API uses `int` for the size of the reads, rather than `uint64_t` so, this suggested change is wider than just changing a single method.

Reply:
Gotcha, thanks for the explanation! I think this change looks good then.